### PR TITLE
Easiest problem ever

### PR DESCRIPTION
**Pull request by macaquedev**
contrivialised

### CODE REVIEW

**Comment by macaquedev** [comment]
clearly not the easiest to deploy!! @myst-6

File: solutions/bio1/index.tsx [hunk 1]
```diff
@@ -38,5 +38,7 @@ export * from "./ModernArt";
 export * from "./MuTorere";
 export * from "./AnagramNumbers";
 export * from "./Juggluging";
+export * from "./DistinctPrimeFactorisation";
+export * from "./GoldbachConjecture";
 export * from "./NumberLadder";
-export * from "./DistinctPrimeFactorisation";
\ No newline at end of file
+export * from "./DistinctPrimeFactorisation";
```
Comment (myst-6) [inline solutions/bio1/index.tsx:44]: Is the `export * from "./DistinctPrimeFactorisation";` line duplicated or am I tripping?

**Review by myst-6 — CHANGES_REQUESTED** [review]
jambloat?

**Comment by macaquedev** [comment]
WOW. I am truly an idiot. Let's try again.

**Review by myst-6 — APPROVED** [review]

**Pull request merged**